toolkit.py

```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/env python
+import sys
+import cv2 as cv
+import numpy as np
+import ntpath
+import functools
+import time
+
+
+def timeit(func):
+    """
+    Poors man profiling function to
+    measure time it takes to finish function.
+    """
+    @functools.wraps(func)
+    def newfunc(*args, **kwargs):
+        start_time = time.time()
+        out = func(*args, **kwargs)
+        elapsed_time = time.time() - start_time
+        msg = 'function [{}] finished in {} ms'
+        print(msg.format(func.__name__, int(elapsed_time * 1000)))
+        return out
+    return newfunc
+
+
+class BoardToolkit:
+    def __init__(self, lo_thresh=100, hi_thresh=255):
+        self.lo_thresh = lo_thresh
+        self.hi_thresh = hi_thresh
+
+    def __get_index(self, arr):
+        """
+        Function for returning min, max tuples from
+        our arrays.
+        """
+        max = np.argmax(arr)
+        min = np.argmin(arr)
+        return (min, max)
+
+    def calc_hist(self, image):
```

Comment:
@Foxiy If the code I read below is right and the docsting also does not lie then this function is not named correctly. It does not calculate a histogram but rather gives you the boundaries of the extremes on a histogram it created. I am not sure what you want to do with that but it seems that the functionality should either be decomposed or this function renamed.